app/controllers/report_controller/widgets.rb

```diff
@@ -403,7 +403,7 @@ def widget_set_form_vars
       end
       @edit[:new][:repfilter] = @edit[:rpt].id
     elsif ["r", "c"].include?(@sb[:wtype])
-      @menu = get_reports_menu
+      @menu = get_reports_menu(current_group, "default")
```

Comment:
@himdel do we also need same change here https://github.com/himdel/manageiq-ui-classic/blob/75a4ac7495e750b5a1f1cac267e86ac6de0b0dc8/app/controllers/report_controller/widgets.rb#L384 to fix edit of existing Dashboard Widget

Reply:
Exactly right, thanks! :)

Updating and adding a spec..